windows_registry/README.md

```diff
@@ -0,0 +1,67 @@
+# Windows Registry Integration (Beta)
+
+## Overview
+
+Watch for changes in Windows Registry keys and forward them to Datadog. Enable this integration to:
+
+- Understand system and application health through Windows Registry key values
+- Monitor for unexpected changes impacting security and compliance requirements
```

Comment:
All of this correct. however I would like to point a few minor details which may be useful.

Regarding
```
Watch for changes in Windows Registry keys and forward them to Datadog. Enable this integration to:
```
* I think it specifying Windows Registry without word keys is sufficient. Word keys is not wrong but it may appear a bit more limiting or ambiguous in the sense that we also report changes in key values.
* Technically we watch and report not only for changes but also current state. It is just as important I think. Not sure if it worth to highlight it. For example a customer may want to see values for OS build number. Not changes are reported but customer may see wrong values and act on it.

Regarding 
```
- Understand system and application health through Windows Registry key values
```
May be we should add word `state` to become `health and state`.

Reply:
made some changes based on your suggestions, thanks!